Services/GEV/WBD/tests/gevWBDDataCollectorTest.php

```diff
@@ -217,6 +251,35 @@ public function test_writeErrorToDB($error) {
 		$this->assertEquals(1, $db->called_executed);
 	}
 
+	public function test_stornoSuccess() {
+		$this->row_id = 25;
+		$success = new gevWBDSuccessWPStorno(simplexml_load_string('<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope">'
+												.'<soap:Body>'
+													.'<ns1:putResponse xmlns:ns1="http://erstanlage.stammdaten.external.service.wbd.gdv.de/">'
+														.'<WPStornoRueckgabewert>'
+															.'<WeiterbildungsPunkteBuchungsId>2015-145-1654</WeiterbildungsPunkteBuchungsId>'
+															.'<VermittlerId>20150728-100390-74</VermittlerId>'
+															.'<InterneVermittlerId>21352</InterneVermittlerId>'
+															.'<BeginnErstePeriode>2015-07-28T00:00:00+02:00</BeginnErstePeriode>'
+														.'</WPStornoRueckgabewert>'
+													.'</ns1:putResponse>'
+												.'</soap:Body>'
+											.'</soap:Envelope>'
+									),$this->row_id);
+
+		$this->data_collector->successStornoRecord($success);
+
+		$needed_case_id = array('usr_id' => 21352
+							  , 'crs_id' => 10);
```

Comment:
Ok. Um Klarheit zu schaffen könntest du die usr_id oben im Test als Variable hinschreiben und dann sowohl im XML einfügen als auch unten in dem Test-Array. Die crs_id könntest du auf dem Mock explizit setzen. Das macht den Test besser verständlich. Statt "needed_" dann "expected_", weil das die erwarteten Daten sind.